controller/src/controller/config.py

```diff
@@ -498,6 +499,135 @@ class EnabledFileserverConfig(FileserverConfig):
     ] = []
 
 
+class FSAdminConfig(BaseModel):
+    """Configuration for filesystem administration environment."""
+
+    model_config = ConfigDict(alias_generator=to_camel, populate_by_name=True)
+
+    affinity: Annotated[
+        Affinity | None,
+        Field(
+            title="Affinity rules",
+            description="Node and pod affinity rules for file server pods",
+        ),
+    ] = None
+
+    application: Annotated[
+        str | None,
+        Field(
+            title="Argo CD application",
+            description="Argo CD application into which to put fsadmin pods",
+        ),
+    ] = None
+
+    extra_annotations: Annotated[
+        dict[str, str],
+        Field(
+            title="Extra annotations",
+            description=(
+                "Extra annotations to add to all user file server ``Job`` and"
+                " ``Pod`` Kubernetes resources"
+            ),
+        ),
+    ] = {}
+
+    extra_volumes: Annotated[
+        list[VolumeConfig],
+        Field(
+            title="Extra volumes",
+            description=(
+                "Additional volumes available to mount inside the fsadmin"
+                " pod. Inclusion in this list does not mean that they will"
+                " be mounted. They must separately be listed under"
+                " ``extravolumeMounts``."
+            ),
+        ),
+    ] = []
+
+    extra_volume_mounts: Annotated[
+        list[VolumeMountConfig],
+        Field(
+            title="Extra mounted volumes",
+            description=(
+                "Additional volumes to mount inside the fsadmin pod."
+            ),
+        ),
+    ] = []
+
+    # We default this so we don't need a configuration-file flag day synced
+    # to the nublado updates.
+    image: Annotated[
+        ContainerImage,
+        Field(
+            title="fsadmin Docker image",
+            description="Docker image to run as fsadmin",
+        ),
+    ] = ContainerImage(
+        repository="ghcr.io/lsst-sqre/nublado-fsadmin",
+        pull_policy=PullPolicy.IF_NOT_PRESENT,
+        tag="8.13.0",
+    )
```

Comment:
Defaults should be in `values.yaml` since that's where we update the version number. This should just be a required configuration parameter with no default.

Reply:
When we do that, anyone who naively upgrades the Nublado controller, without changing their values.yaml in Phalanx, breaks their system.

I can do that (and make this a major version increase) but it was an intentional choice to default these so that people's extant configs would keep working.